recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
+
+build:
+    skip: True  # [py3k]
```

Comment:
OK -- as this recipe is specifcally for conda-forge, in its present state, then yes, that makes sense.

stay tuned for an update...